projects/origin-stable/release.yaml

```diff
@@ -65,8 +65,8 @@ spec:
     from: {"kind":"DockerImage","name":"docker.io/openshift/origin-release:nodejs-8-browser-tests"}
     referencePolicy: {"type": "Local"}
     importPolicy: {"scheduled": true}
-  - name: tectonic-console-builder-v19
-    from: {"kind":"DockerImage","name":"quay.io/coreos/tectonic-console-builder:v19"}
```

Comment:
Don't remove v19, that will break release-4.4 PRs

Reply:
Ok, do I add a new one for v20 then?

Reply:
> Ok, do I add a new one for v20 then?

Yes, v19 will be used for 4.4, and v20 will be used for 4.5+. So we need both.

Reply:
ok, fixed, ty